CMake/Dependencies/libmbedtls-CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+cmake_minimum_required(VERSION 2.8)
+
+project(libmbedtls-download NONE)
+
+include(ExternalProject)
+
+set(BUILD_SHARED 1)
+if (BUILD_STATIC)
+	set(BUILD_SHARED 0)
+endif()
+
+set(CMAKE_MACOSX_RPATH 0)
+if (${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
+    set(CMAKE_MACOSX_RPATH 1)
+endif()
+
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -DMBEDTLS_SSL_DTLS_SRTP")
+ExternalProject_Add(
+  project_libmbedtls
+  " TODO: needs to replace this with official mbedTLS branch
+  "       when they add DTLS-SRTP extension support.
+  " Reference: https://github.com/ARMmbed/mbedtls/pull/3235
+  GIT_REPOSITORY 	https://github.com/lherman-cs/mbedtls.git
```

Comment:
I forked the PR branch for the DTLS-SRTP extension support. The main reason is so that the branch is persisted. Otherwise, there might be a new update to the branch, or the PR gets merged and the branch gets deleted.